lib/nx_zlib.c

```diff
@@ -622,6 +622,27 @@ static void nx_close_all()
 	return;
 }
 
+#define SYSFS_GZIP_CAPS "/sys/devices/vio/ibm,compression-v1/nx_gzip_caps/"
+static int nx_query_job_limits()
+{
+	char fname[64], buf[32];
+	ssize_t cnt;
```

Comment:
Is this variable used anywhere?

Reply:
@rzinsly Nope. And neither is fname. These are leftovers from the first implementation. Fixed now.